department-of-veterans-affairs/caseflow
# Autosave failure message

Connects #3377 

The success message will be implemented in a later PR.

To test:
1. Go to the hearing worksheet
1. Update line 263 in Dockets.js so that the url is incorrect
1. Update a field in the hearing worksheet
1. Confirm the error message is displayed
1. Fix line 263 in Dockets.js
1. Confirm that the save goes through and the error message goes away
